### ISSUE TITLE
Add reminder about Jira notification email address

### ISSUE THREAD

**Issue by daniel-beck** [jenkins-infra/helpdesk#3204]
### Service(s)

Accounts

### Summary

It's not uncommon for maintainers to use a different (usually outdated/obsolete) email address on Jira compared to the account app. This creates friction when trying to inform maintainers about vulnerabilities.

I don't remember whether it was a deliberate choice to not override the Jira notification email address with the one from the account app, but it might be. I know I deliberately did that for a while.

Unless we want to (and can) change this behavior, an easy first step would be to add a reminder to the account app that the Jira notification email address needs to be changed separately.

**Comment by timja** [comment]
I think it would be better to just update from ldap on login?

Would need to modify this:
![image](https://user-images.githubusercontent.com/21194782/197724832-336fd0fe-73ac-46ae-ae88-22a704064780.png)

**Comment by daniel-beck** [comment]
> I think it would be better to just update from ldap on login?

Right, I explicitly mention that as a possibility. The problem with that is that it could override good Jira values with bad account app values. We should make sure that's not the case first. For example, around 70 LDAP accounts have @java.net email addresses. If they have a better email address in Jira, we'd break that.

**Comment by daniel-beck** [comment]
I think this issue is obsolete now. Could someone confirm?
